games-fps/prboom-plus/prboom_plus-2.5.1.4.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Enhanced clone of the classic first-person shooter Doom"
+DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom
+ project by Andrey Budko"
```

Comment:
```diff
-DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom
- project by Andrey Budko"
+DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom \
+project by Andrey Budko"
```

This is the preferred way to wrap lines in HaikuPorts.

Reply:
You can use freedoom to test it out: https://freedoom.github.io/download.html